apps/user/src/components/form/ProfileUploader/ProfileUploader.hook.ts

```diff
@@ -116,20 +116,28 @@ export const useProfileUploader = (
     [processImageFile]
   );
 
-  const refreshProfileImage = useCallback(
-    (file?: File) => {
-      if (file) {
-        refresh(file, {
-          onSuccess: (newDownloadUrl) => handleUploadSuccess(newDownloadUrl),
-          onError: () => onPhotoUpload(false),
-        });
-      } else if (!isUploading && data?.status !== 'REJECTED') {
-        const storedImageUrl = Storage.getItem('downloadUrl');
-        if (storedImageUrl) setImgSrc(storedImageUrl);
-      }
-    },
-    [isUploading, refresh, data?.status, handleUploadSuccess, onPhotoUpload]
-  );
+  const refreshProfileImage = useCallback(() => {
+    if ((!isUploadPictureStored && !isUploading) || data?.status === 'REJECTED') {
+      refresh(undefined, {
+        onSuccess: (newDownloadUrl) => handleUploadSuccess(newDownloadUrl),
+        onError: () => onPhotoUpload(false),
+      });
+    } else {
+      const storedImageUrl = Storage.getItem('downloadUrl');
+      if (storedImageUrl) setImgSrc(storedImageUrl);
+      refresh(undefined, {
+        onSuccess: (newDownloadUrl) => handleUploadSuccess(newDownloadUrl),
+        onError: () => onPhotoUpload(false),
+      });
+    }
```

Comment:
반복되는 로직 함수화 해도 괜찮을 것 같아요
```suggestion
  const refreshProfileImage = useCallback(() => {
    const handleRefresh = () => {
    refresh(undefined, {
      onSuccess: (newDownloadUrl) => handleUploadSuccess(newDownloadUrl),
      onError: () => onPhotoUpload(false),
    });
  };
  
    if ((!isUploadPictureStored && !isUploading) || data?.status === 'REJECTED') {
      handleRefresh()
    } else {
      const storedImageUrl = Storage.getItem('downloadUrl');
      if (storedImageUrl) setImgSrc(storedImageUrl);
      handleRefresh()
    }
```

이런 느낌으로...?